app/portainer/views/stacks/edit/stack.html

```diff
@@ -94,6 +94,13 @@
             <uib-tab-heading> <i class="fa fa-pencil-alt space-right" aria-hidden="true"></i> Editor </uib-tab-heading>
             <form class="form-horizontal" ng-if="state.showEditorTab" style="margin-top: 10px;">
               <div class="form-group">
+                <span class="col-sm-12 text-muted small" ng-if="stackType == 2 && composeSyntaxMaxVersion == 2">
+                  This stack will be deployed using the equivalent of <code>docker-compose</code>. Only Compose file format version <b>2</b> is supported at the moment.
+                  <br /><br />
```

Comment:
I know we use double `br` in createstack.html, but it's a bad practice. can we please just replace it in both files with a small bottom margin?